src/v2/Apps/Order/Utils/shippingUtils.ts

```diff
@@ -120,12 +120,15 @@ export const convertShippingAddressForExchange = (
 }
 
 export const convertShippingAddressToMutationInput = (
+  phoneNumber: SavedAddressType["phoneNumber"],
   address: SavedAddressType
 ): UserAddressAttributes => {
+  const modifiedAddress = assign(address, { phoneNumber })
```

Comment:
Can this be done with something like `const modifiedAddress = { ...address, phoneNumber }`?

Reply:
Good call!